zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/AngularObject.java

```diff
@@ -22,26 +22,33 @@
 import java.util.Objects;
 import java.util.concurrent.ExecutorService;
 
+import org.apache.zeppelin.interpreter.remote.RemoteAngularObject;
 import org.apache.zeppelin.scheduler.ExecutorFactory;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.xml.bind.annotation.*;
+
 /**
  * AngularObject provides binding between back-end (interpreter) and front-end
  * User provided object will automatically synchronized with front-end side.
  * i.e. update from back-end will be sent to front-end, update from front-end will sent-to backend
  *
  * @param <T>
  */
+@XmlRootElement(name = "AngularObject")
+@XmlSeeAlso({RemoteAngularObject.class})
+@XmlAccessorType(XmlAccessType.NONE)
 public class AngularObject<T> {
+  @XmlElement
   private String name;
+  @XmlElement
   private T object;
-  
   private transient AngularObjectListener listener;
-  private transient List<AngularObjectWatcher> watchers
-    = new LinkedList<AngularObjectWatcher>();
-  
-  private String noteId;   // noteId belonging to. null for global scope 
```

Comment:
When you make changes to existing code (and write a new code as well) you always have to think about a person who is going to read it later.

It may be you, in a few years later, or just a reviewer but it's you who can either simplify their life, or complicate it.
I.e changes like re-ordering lines and adding\removing whitespaces makes life of reviewer harder as he now will need to ges your (author's) intent (why did you do it? is it crucial for your program, so it can work correctly?).

So generally safest way is to touch as less code as required to implement the feature. And keep all refactoring and improvements in the separate PRs that can be clearly marked as such and reviewed separately.